<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,3,3,3,3,3,3,3,3,3,3,3,3" columnstretch="1,1,1,3,2,2">
    <item row="0" column="0" rowspan="9" colspan="4">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>New element:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>x:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="x"/>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>y:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="y"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Height:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="height"/>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Lenght:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="lenght"/>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Width:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="width"/>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Radius:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLineEdit" name="radius"/>
    </item>
    <item row="7" column="5">
     <widget class="QCheckBox" name="isDish">
      <property name="text">
       <string>Dish</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QPushButton" name="addElement">
      <property name="text">
       <string>Add element</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QPushButton" name="PopButton">
      <property name="text">
       <string>Pop element</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="ClearButton">
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Path to data:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="5">
     <widget class="QLineEdit" name="path"/>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QCheckBox" name="isNeedToRead">
      <property name="text">
       <string>Read from this path</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2" colspan="2">
     <widget class="QPushButton" name="LoadDumpData">
      <property name="text">
       <string>Load/dump data</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
